exoctk/exoctk_app/templates/contam_visibility.html

```diff
@@ -60,42 +60,43 @@ <h1>Contamination & Visibility Calculator</h1>
             <div class="col-sm-10">
                 
                 <div class="input-group">
-                  <div class="input-group-addon" style='width:120px'>\(T_\text{eff}\)</div>
+                  <div class="input-group-addon" style='width:60px'>R.A.</div>
                       {{ form.ra(value=form.ra.data, size=10, rows=1, class='form-control') }}
                   <div class="input-group-addon" style='width:60px'>Decimal Degrees</div>
                 </div>
                 <span id="helpBlock" class="help-block">The right ascension of the target</span>
                 <br>
 
                 <div class="input-group">
-                  <div class="input-group-addon" style='width:120px'>\(\log (g)\)</div>
+                  <div class="input-group-addon" style='width:60px'>Decl.</div>
```

Comment:
for some reason `contam_visibility.resolve` has been returning decimal degrees instead of RA/Dec. is this meant to be decimal degrees?

Reply:
We aren't using `contam_visibility.resolve` anymore though, right? It's the results from `utils.get_target_data` now, which is in decimal degrees. Is that problematic?

Reply:
So `utils.get_target_data` outputs decimal degrees, so it should be `<div class="input-group-addon" style='width:60px'>Decimal Degrees</div>` , I think that's the last thing that requires changing

Reply:
whoops nvm